src/types.h

```diff
@@ -337,6 +337,8 @@ typedef struct {
 	bool manage;
 	bool focus;
 	bool border;
+	bool forced_rect;
```

Comment:
Instead of these two attributes, I would rather go with just `xcb_rectangle_t *rect`.